.github/workflows/build_and_test.yml

```diff
@@ -70,6 +70,18 @@ jobs:
       with:
         name: artefacts
         path: wheelhouse/
+    - name: Install docs dependencies
+      if:  (matrix.os == 'ubuntu-20.04') && (github.event_name == 'pull_request')
+      run: |
+        pip install -r .github/workflows/docs/requirements.txt
+    - name: Build docs
+      if:  (matrix.os == 'ubuntu-20.04') && (github.event_name == 'pull_request')
+      timeout-minutes: 20
+      run: |
+        cd .github/workflows/docs
+        mkdir extensions
+        ./build-docs -a -i -d ${GITHUB_WORKSPACE}/.github/workflows/docs/extensions
```

Comment:
The `build-docs` script here could be tidied up a bit: the `-a` option can be removed.

Reply:
I have cleaned up the file, will need to check if the upload still works after merging this in a RC release.

Reply:
(have removed the i option, too)